ext/shopify-cli/extconf.rb

```diff
@@ -0,0 +1,27 @@
+require 'rbconfig'
+require 'fileutils'
+
+gem = File.expand_path('../../../', __FILE__)
+exe = File.join(gem, 'bin', 'shopify-cli')
+script = exe + '.sh'
+symlink = '/usr/local/bin/shopify-cli'
+
+script_content = <<~SCRIPT
+  #!/usr/bin/env bash
+  #{RbConfig.ruby} --disable=gems -I #{gem} #{exe} $@
+SCRIPT
```

Comment:
Just out of curiosity: I made a similar change for the packaging tasks, but I create template files that I copy and edit instead of writing them out from ruby. Is there a preferred way between this and that?

Reply:
🤷  I don't know if there's a preferred way!